source/adapters/cuda/usm_p2p.cpp

```diff
@@ -13,28 +13,24 @@
 
 UR_APIEXPORT ur_result_t UR_APICALL urUsmP2PEnablePeerAccessExp(
     ur_device_handle_t commandDevice, ur_device_handle_t peerDevice) {
-
-  ur_result_t result = UR_RESULT_SUCCESS;
   try {
     ScopedContext active(commandDevice->getContext());
     UR_CHECK_ERROR(cuCtxEnablePeerAccess(peerDevice->getContext(), 0));
   } catch (ur_result_t err) {
-    result = err;
+    return err;
   }
-  return result;
```

Comment:
These changes appear unrelated. Please submit them as a separate cleanup

Reply:
Because of the overhead involved in getting UR PRs merged I would be inclined to make an exception and keep these changes in this PR. They will still appear as a separate commit since we don't squash in UR

Reply:
That's not a good reason. The hardest bug I ever had to solve was because of "just an extra bit of code" unrelated to the commit message.

Reply:
I defer to your judgement

Reply:
New PR for those format changes is here: https://github.com/oneapi-src/unified-runtime/pull/1311